src/main/java/com/salesforce/storm/spout/dynamic/metrics/LogRecorder.java

```diff
@@ -43,12 +42,11 @@
 public class LogRecorder implements MetricsRecorder {
 
     private static final Logger logger = LoggerFactory.getLogger(LogRecorder.class);
-    private final Map<String, Long> counters = Maps.newConcurrentMap();
-    private final Map<String, CircularFifoQueue> averages = Maps.newConcurrentMap();
```

Comment:
Was averages just not used?

Reply:
Correct, which made this one easy to deal with. 😎